frontend/packages/dev-console/src/components/pipelineruns/PipelineTaskLogs.tsx

```diff
@@ -129,38 +132,59 @@ class PipelineTaskLogs extends React.Component<PipelineTaskLogsProps, PipelineTa
     const { obj, taskName } = this.props;
     const { renderContainers, fetching, targetHeight } = this.state;
     const containerStatus = _.get(this.props, ['obj', 'data', 'status', 'containerStatuses'], []);
+    const downloadLogs = () => {
+      const logString = this.scrollPane.current.innerText
```

Comment:
Interesting idea! Download is that we can still be loading the data. But I see that `resource-log` suffers the same fate. Just less chance of error due to a single stream.